### PR TITLE
Feature additions for WRC Analytics upload

### PR DESCRIPTION
**Pull request by hk4**
Features added:
- PDF export added
- Email enhancements through HTML encoding of automated email response
- Embedded pivot table into automated email response
- Link generation to access Subscription Manager through email

### CODE REVIEW

File: PivotSubscriptions/Task.cls [hunk 1]
```diff
@@ -10,7 +10,9 @@ Method OnTask() As %Status
 	// Construct current day and time for query
 	Set tDayOfWeek=$zd($h,10)
 	Set tHour=$p($zt($piece($h,",",2),2),":",1)
+	Set tMinute=$p($zt($piece($h,",",2),2),":",2)
```
Comment (psteiwer) [inline PivotSubscriptions/Task.cls]: Looks like this line was duplicated

File: PivotSubscriptions/Task.cls [hunk 2]
```diff
@@ -56,20 +122,13 @@ ClassMethod SendEmail(pPivot As %String, pEmails As %String, pFormat As %String)
 	
 	// Only generate file if requested pivot has results
 	If tHasResults {
-		Set tTitle = $E(pPivot,1,*-6)
-		Set pParms("TITLE")=tTitle
+		Set pParms("TITLE")=$E(pPivot,1,*-6)
 		Set pParms("SUBTITLE")="Report generated on "_$zd($h,6)
 		Set pParms("MDX")=mdx
 		
 		// if based on if pdf or excel is selected, change filename accordingly
```
Comment (psteiwer) [inline PivotSubscriptions/Task.cls]: I think we can take out this comment if the code no longer does this

File: PivotSubscriptions/UI/SubscriptionManager.cls [hunk 3]
```diff
@@ -79,6 +79,17 @@ ClientMethod onPopupAction(popupName, action, value) [ Internal, Language = java
 	}
 }
 
+/// Access edit subscription independently
+ClientMethod unsubscribe(subscriptionId,pivotName) [Internal, Language = javascript ]
```
Comment (psteiwer) [inline PivotSubscriptions/UI/SubscriptionManager.cls]: Should this method be included in this version? Is it used at all?

**Comment by hk4** [comment]
Added changes requested by Peter

**Pull request merged**